README.md

```diff
@@ -93,7 +93,7 @@ Znaczy [pull requestów](https://github.com/nurkiewicz/polski-w-it/pulls), nie t
 | process (_v_)        | procesować          | przetwarzać                           |
 | progress             |                     | postęp                                |
 | property             |                     | właściwość, opcja, cecha              |
-| proxy                | proksy              |                                       |
+| proxy                | proksy              | serwis pośredniczący                  |
```

Comment:
Teraz pomyślałem, że może jednak `serwer pośredniczący` 🤔

Reply:
um what?

Reply:
No bardziej serwer niż serwis. Nvm.

Reply:
serwis jest bardziej generyczny niz serwer, przyklad:
- jak masz robic proxy pass w nginx to bedzie to serwer (albo jedna instancja z wielu)
- ale jak juz masz takie istio to tam po drodze jest kilka innych serwerow (jakies terninacje TLS, load balancing, mozna dorzucic jakis sidecar z dodattkowymi rzeczami jak duplikacja requestow na potrzeby logowania itd) ktore skladaja sie na calosc proxy.

Ogolnie chodzi mi o to ze proxy moze byc warstwą i w srodku mogą być czaaaary - vide nasz ulubiony projekt ese-device ......... opentsdb, a po srodku jest kilka dziwnych rzeczy ;)